src/GetHtml.php

```diff
@@ -7,8 +7,10 @@
 class GetHtml implements UrlParser
 {
     /**
+     *
      * Get the URLs html.
      *
+     * @param string $url
      * @return string
      */
     public function getHtml(string $url): string
```

Comment:
The method should not contain type hints as this prevents other implementations from using representations of url and html that are not strings.